internal/controller/numaresourcesscheduler_controller.go

```diff
@@ -58,12 +59,24 @@ import (
 	"github.com/openshift-kni/numaresources-operator/pkg/status"
 )
 
+const (
+	// ActivePodsResourcesSupportSince defines the OCP version which started to support the fixed kubelet
+	// in which the PodResourcesAPI lists the active pods by default
+	activePodsResourcesSupportSince = "4.20.999"
```

Comment:
This version value should be modified per Ystream branch (main is 4.20.z, release-4.19 will use 4.19.z where z is the first supported stream in this minor release.)